src/main/g8/src/main/scala/$package$/HttpServer.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+package $package$
+
+// #http-server
+import akka.actor.typed.Behavior
+import akka.actor.typed.PostStop
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.adapter._
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.Http.ServerBinding
+import akka.http.scaladsl.server.Route
+
+import scala.util.Failure
+import scala.util.Success
+
+object HttpServer {
```

Comment:
Do we really need this actor? Does it provide anything above what the `ServerBinding` already provides?

Reply:
Yeah, not 100% sure. I think the strongest argument I have is that it keeps the bootstrap behavior dealing with the same abstraction level (starting a few actors, watching them in case of crash, injecting refs between). Rather than mixing a little bit of binding setup futures and callbacks with starting other parts of the application (the registry actor), etc in the bootstrap behavior. 

I guess it depends on if we want this to be a starting point for building an actor system with a HTTP frontend (where this kind of separation of concerns makes more sense), or just a minimal HTTP starting point (where it is perhaps too much boiler plate). If we want minimal starting point we should maybe scrap the whole interact-with-actors part completely?